app/vue/.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: {
+        '@typescript-eslint/explicit-function-return-type': 'error',
```

Comment:
Made the Vue app as strict as I could for typescript, to catch everything, can revert these if wanted.